doc/host_create.md

```diff
@@ -263,13 +268,14 @@ cluster               Cluster ID from VMware
 resource_pool         Resource Pool ID from VMware
 path                  Path to folder
 guest_id              Guest OS ID form VMware
-scsi_controller_type  ID of the controller from VMware
 hardware_version      Hardware version ID from VMware
 add_cdrom             Must be a 1 or 0, Add a CD-ROM drive to the virtual machine
 cpuHotAddEnabled      Must be a 1 or 0, lets you add memory resources while the machine is on
-memoryHotAddEnabled   Must be a 1 or 0, lets you add CPU resources while the machine is on
-start                 Must be a 1 or 0, whether to start the machine or not
```

Comment:
why did you remove start from here? it should still be here in when creating a host.